docker-compose.yml

```diff
@@ -56,7 +69,22 @@ services:
       - "traefik.http.routers.${COMPOSE_PROJECT_NAME}-varnish.tls=true"
       - "traefik.http.services.${COMPOSE_PROJECT_NAME}-varnish.loadbalancer.server.port=6081"
       - "traefik.docker.network=stonehenge-network"
-
+  # robo:
```

Comment:
what is this for?

Reply:
Hel.fi platform has robot framework testing but we don't. I thought it's best to comment out